src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -171,15 +172,24 @@ Copyright (c) .NET Foundation. All rights reserved.
          in some cases. -->
     <NETSdkError Condition="'$(SelfContained)' != 'true'" ResourceName="ILLinkNotSupportedError" />
 
-    <PropertyGroup>
-      <ILLinkWarningLevel Condition=" '$(ILLinkWarningLevel)' == '' And
-                                      '$(EffectiveAnalysisLevel)' != '' And
+    <PropertyGroup Condition=" '$(ILLinkWarningLevel)' == '' ">
+      <ILLinkWarningLevel Condition=" '$(EffectiveAnalysisLevel)' != '' And
                                        $([MSBuild]::VersionGreaterThanOrEquals($(EffectiveAnalysisLevel), '5.0')) ">5</ILLinkWarningLevel>
       <ILLinkWarningLevel Condition=" '$(ILLinkWarningLevel)' == '' ">0</ILLinkWarningLevel>
+    </PropertyGroup>
+
+    <PropertyGroup>
       <ILLinkTreatWarningsAsErrors Condition=" '$(ILLinkTreatWarningsAsErrors)' == '' ">$(TreatWarningsAsErrors)</ILLinkTreatWarningsAsErrors>
       <_ExtraTrimmerArgs>--skip-unresolved true $(_ExtraTrimmerArgs)</_ExtraTrimmerArgs>
       <TrimMode Condition=" '$(TrimMode)' == '' ">copyused</TrimMode>
     </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(_TrimmerDefaultAction)' == '' ">
```

Comment:
Who's configuring this if it's not empty?

Reply:
Removed. I had originally made it public, but I don't think we want to expose it as a user knob yet.